feature/experimental/bgp/ate_tests/bgp_remove_private_as/README.md

```diff
@@ -0,0 +1,41 @@
+# RT-1.11: BGP remove private AS 
+
+## Summary
+
+BGP remove private AS
+
+## Procedure
+
+*   Establish BGP sessions as follows between ATE and DUT.
+    *   ATE emulates two eBGP neighbors peering with the DUT using public AS numbers.
+        *   DUT Port1 (AS 500) ---eBGP--- ATE Port1 (AS 100)
+        *   DUT Port2 (AS 500) ---eBGP--- ATE Port2 (AS 200)
+    *   Inject routes with AS_PATH modified to have private AS number 65501 from eBGP neighbor #1 
+        (ATE Port1).
+    *   Validate received routes on ATE Port2 should have AS Path "500 100 65501".
+    *   Configure "remove private AS" on DUT.    
```

Comment:
Just to be explicit, can we add that remote-private-as is configured in REMOVE_ALL mode?
thanks,

Reply:
updated README, please check.